examples/end-to-end-examples/support_chatbot/README.md

```diff
@@ -0,0 +1,179 @@
+
+**Fields to replace with your values:** `REPLACE_WITH_YOUR_CLUSTER` (in the command), `CLOUD` (in the yaml), `BUCKET_NAME` (in the yaml), `CHECKPOINT_FOLDER_NAME` (in the yaml), `HF_FOLDER_NAME` (in the yaml)
+
+**Inputs:** the final checkpoint from step 4 saved to the `save_folder` specified in the previous yaml and where you want the converted checkpoint to go in `hf_output_path`
+
+**Command:**
+```bash
+mcli run -f mcli-yamls/convert_checkpoint_to_huggingface.yaml --cluster REPLACE_WITH_YOUR_CLUSTER
+```
+
+**Outputs:** the ``CLOUD://BUCKET_NAME/support-bot-demo/converted_checkpoints/HF_FOLDER_NAME`` folder, containing the HuggingFace checkpoint files
+
+## Step 6: Deploy your model
+
+Now that we have our trained model, we will deploy it using MosaicML inference. This will allow us to use the model as an API. For more examples of inference deployments, see [inference-deployments](../../inference-deployments/)
+
+
+**Fields to replace with your values:** `REPLACE_WITH_YOUR_CLUSTER` (in the command), `BUCKET_NAME` (in the yaml), `HF_FOLDER_NAME` (in the yaml)`
+
+**Inputs:** the HuggingFace format checkpoint from step 5
+
+**Command**:
+```bash
+mcli deploy -f mcli-yamls/deploy_llm.yaml --cluster REPLACE_WITH_YOUR_CLUSTER
+```
+
+**Outputs:** A deployment for the language model
+
+
+## Step 7: Application with gradio
+
+Now that we've processed our data, trained our models, and deployed our models, we can run the application! We will use Gradio and LangChain to make a simple question answering interface.
+
+We will make use of the MosaicML integration in LangChain for [LLMs](https://github.com/hwchase17/langchain/blob/master/langchain/llms/mosaicml.py) and [embeddings](https://github.com/hwchase17/langchain/blob/master/langchain/embeddings/mosaicml.py), and use the [`RetrievalQA`](https://python.langchain.com/en/latest/modules/chains/index_examples/vector_db_qa.html?highlight=retrievalqa) abstraction with the [`FAISS`](https://python.langchain.com/en/latest/modules/indexes/vectorstores/examples/faiss.html?highlight=faiss) to run the application locally.
+
+Upon the first run of this app, expect that it might take ~20 minutes to embed all of the data and store the vector store in `retrieval_data/vectors.pickle`. After the first run, unless you delete the pickle file, it will just reference the the pickle file and not re-embed.
```

Comment:
```suggestion
Upon the first run of this app, expect that it might take ~20 minutes to embed all of the data and store the vector store in `retrieval_data/vectors.pickle`. After the first run, unless you delete the pickle file, it will just reference the pickle file and not re-embed.
```